tap_jira/streams.py

```diff
@@ -192,6 +192,8 @@ def sync(self):
                   "expand": "changelog,transitions",
                   "validateQuery": "strict",
                   "jql": jql}
+        response = Context.retrieve_timezone()
+        LOGGER.info(response)
```

Comment:
This gives us logs like
```sh
2022-02-15 18:40:32,221Z    tap - INFO HTTP request to "test" endpoint took 0.083s, returned status code 200
2022-02-15 18:40:32,359Z    tap - INFO America/New_York
2022-02-15 18:40:32,605Z    tap - INFO HTTP request to "issues" endpoint took 0.245s, returned status code 200
```
Why do we need this?

Reply:
This request is also a duplicate of line 187

Reply:
This Logger message was initially meant to capture the timing of different streams, but doesn't make sense anymore. Also the duplicate of context.retrieve_timezone() is removed on line 195